config/puma.rb

```diff
@@ -0,0 +1,38 @@
+# Puma can serve each request in a thread from an internal thread pool.
+# The `threads` method setting takes two numbers: a minimum and maximum.
+# Any libraries that use thread pools should be configured to match
+# the maximum value specified for Puma. Default is set to 5 threads for minimum
+# and maximum; this matches the default thread size of Active Record.
+#
+max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
```

Comment:
```suggestion
max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 1 }
```

So we can use pry more easily in development

Reply:
I propose to open PR on rails-template for this also 👍 👎 ?